.github/workflows/validate-commit-message.py

```diff
@@ -12,9 +12,10 @@
 
 
 def prefix_invalid(prefix):
-    package = prefix.split(':')[0]
-    if package.strip() not in EXISTING_PACKAGES and package not in KEYWORDS:
-        return True
+    packages = prefix.split(':')[0].split(',')
+    for package in packages:
+        if package.strip() not in EXISTING_PACKAGES and package not in KEYWORDS:
```

Comment:
```suggestion
    for package in map(str.strip, packages):
        if package not in EXISTING_PACKAGES and package not in KEYWORDS:
```
Seems that both tests in the `if` should use a stripped version of the keyword?

Reply:
Good catch